### ISSUE TITLE
Where is the source code of SNES9X+Satellaview+

### ISSUE THREAD

**Issue by awsms** [SatellaviewPlus/Website#2]
Hi! I'd like to run this natively on Linux. Since SNES9X is already Linux compatible, could you please provide the patches you've added to SNES9X?
Thank you very much for this super cool project.

**Comment by HowlingFoxRouko** [comment]
Hello. Thank you for your interest. At this time the project is closed-source so we will not be able to provide the source code at this time.

This said however, the client is being upgraded for multi-platform usage as we speak. There are a few more things to work out but at this time there is hope on the horizon. :)

**Comment by XargonWan** [comment]
Greetings, is there any willing to open source this project?

**Comment by HowlingFoxRouko** [comment]
> Greetings, is there any willing to open source this project?

Hello and thank you for your reply. At this time the project is closed-source only.

**Comment by XargonWan** [comment]
> > Greetings, is there any willing to open source this project?
> 
> Hello and thank you for your reply. At this time the project is closed-source only.

Are you sure that the original emulators licences allows a closed source redistribution?

**Comment by HowlingFoxRouko** [comment]
Given that this is not a part of the emulator distribution and is only generating data for the end user's emulator, we're well within our rights to close the source.

**Comment by HowlingFoxRouko** [comment]
> > > Greetings, is there any willing to open source this project?
> > 
> > Hello and thank you for your reply. At this time the project is closed-source only.
> 
> Are you sure that the original emulators licences allows a closed source redistribution? 

Given that this is not a part of the emulator distribution and is only generating data for the end user's emulator, we're well within our rights to close the source.

**Comment by XargonWan** [comment]
> > > > Greetings, is there any willing to open source this project?
> > > 
> > > 
> > > Hello and thank you for your reply. At this time the project is closed-source only.
> > 
> > 
> > Are you sure that the original emulators licences allows a closed source redistribution?
> 
> Given that this is not a part of the emulator distribution and is only generating data for the end user's emulator, we're well within our rights to close the source.

Ah I see it's distributed separately?
Well best luck for this project, can't wait to test it on linux (on Lutris it fails for some reason).
I hope that you will decide to open source it one day :)

**Comment by HowlingFoxRouko** [comment]
> > > > > Greetings, is there any willing to open source this project?
> > > > 
> > > > 
> > > > Hello and thank you for your reply. At this time the project is closed-source only.
> > > 
> > > 
> > > Are you sure that the original emulators licences allows a closed source redistribution?
> > 
> > 
> > Given that this is not a part of the emulator distribution and is only generating data for the end user's emulator, we're well within our rights to close the source.
> 
> Ah I see it's distributed separately? 
Yes; the client itself is a separate, closed-source program, so it is distributed separately. 😁
>Well best luck for this project, can't wait to test it on linux (on Lutris it fails for some reason). 
Thank you so much! We are actively working on a client update that should bring new platform capabilities, so hopefully Lutris isn't needed and it will natively run on Linux. (We hope!!)
>I hope that you will decide to open source it one day :)
Someday, perhaps, but for now we've made the choice to go closed-source. 😄
